postdata.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+$colloquial_name_input = $_POST['colloquial_name'];
+$latin_name_input = $_POST['latin_name'];
+$size_input = $_POST['size_cm'];
+$image_input = $_POST['image_url'];
+
+
+$colloquial_name_sanitized = filter_var($colloquial_name_input, FILTER_SANITIZE_STRING);
+$latin_name_sanitized = filter_var($latin_name_input, FILTER_SANITIZE_STRING);
+$size_sanitized = filter_var($size_input, FILTER_SANITIZE_NUMBER_INT);
+$image_sanitized = filter_var($image_input, FILTER_SANITIZE_URL);
+
+
```

Comment:
can this be functionalised and added to the functions.php doc? Could use && to add all conditions together and output 'valid' or 'invalid'.

If functionalised and moved to functions.php doc, include dock block.

Reply:
functionalised and unit tested